tests/python/gaia-ui-tests/gaiatest/tests/functional/camera/test_front_camera_take_photo_with_timer.py

```diff
@@ -0,0 +1,49 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.camera.app import Camera
+from gaiatest.apps.camera.app import ImagePreview
+
+
+class TestCamera(GaiaTestCase):
+
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+
+        # Turn off geolocation prompt
+        self.apps.set_permission('Camera', 'geolocation', 'deny')
+
+    def test_capture_a_photo(self):
+        """https://moztrap.mozilla.org/manage/case/15800/"""
+        self.previous_number_of_pictures = len(self.data_layer.picture_files)
+
+        self.camera = Camera(self.marionette)
+        self.camera.launch()
+
+        # Open Camera Settings and turn on timer
+        self.camera.tap_menu_button()
```

Comment:
Regarding the comment above, the name of this function can be improved I think. How about: 
```python
camera.tap_settings_button()
```